devops/actions/e2e-tests-win/action.yml

```diff
@@ -0,0 +1,45 @@
+name: 'Run SYCL End-to-End testing'
+description: 'Run SYCL End-to-End testing'
+inputs:
+  sycl_artifact:
+    description: 'Name of the artifact, that contains compiler toolchain'
+    required: true
+  targets:
+    description: 'List of SYCL backends with set of target devices per each to be tested iteratively'
+    required: true
+  cmake_args:
+    description: 'Extra arguments to cmake command'
+    required: false
+
+post-if: false
+runs:
+  using: "composite"
+  steps:
+  - name: Download compiler toolchain
+    uses: actions/download-artifact@v3
+    with:
+      name: ${{ inputs.sycl_artifact }}
+  - name: Extract SYCL toolchain
+    shell: bash
+    run: |
+      mkdir install
+      tar -xf llvm_sycl.tar.gz -C install
+      rm llvm_sycl.tar.gz
+  - name: Configure
+    shell: cmd
+    run: |
+      mkdir build-e2e
+      set PATH=%GITHUB_WORKSPACE%\install\bin;%PATH%
+      cmake -GNinja -B build-e2e -S.\llvm\sycl\test-e2e -DSYCL_TEST_E2E_TARGETS="${{ inputs.targets }}" -DCMAKE_CXX_COMPILER="clang++" -DLLVM_LIT="..\llvm\llvm\utils\lit\lit.py" ${{ inputs.cmake_args }}
```

Comment:
I'm not sure if we want `clang++` or `clang-cl` here.

Reply:
Previously, I did try experimenting with clang-cl instead of clang++. However, I got a lot of compiler errors. My understanding is that, with Git's Bash, clang++ works fine. However, with U4Win Bash, I had to use `clang-cl` to run the tests. IDK why this happens, but that's something I observed while playing around with the CI.